Task 2 te 347 provide web fixt

kbayraktar opened the pull request:
fixture methods added

kpanier committed:
: TE-347 Provide web-fixture for TE 2.0 

Update to latest selenium version

kpanier committed:
TE-347 Provide web-fixture for TE 2.0 

- Ignore blocking test
- No embedded libs anymore, because the tst execution is managed by the
build system
- Basic fixture operations to run a simple googletest

kpanier committed:
TE-347 Provide web-fixture for TE 2.0 

Adding clean method to fixture

kpanier committed:
TE-347 using selections

kpanier committed:
TE-347 using selections

kpanier committed:
Support AngularJS with special fixtures

kpanier committed:
Support AngularJS with special fixtures

kpanier committed:
TE-377 Proxy Support

- Using proxy settings fixture

kpanier committed:
TE-377 Proxy Support

- Logging proxy in webfixture

kpanier committed:
Improve Webfixture

- using Marionette Driver
- Using WebDriverManager to handle downloads.

Commit:
Fixtures expanded for testing angular JS-App.

Commit:
proxy username and password system properties added.

kpanier commented:
I thought we should merge this branch into the task 1 branch and not in development now. The task 1 branch has the purpose of a new implementation of the fixture

Commit:
[TE-347] changes implemented see following entries:
- WebDriverFixture.java - checkEnabled Method returns now Boolean values and not null
- WebDriverFixture.aml label changed
